test/interval_tests/complex.jl

```diff
@@ -44,3 +46,13 @@ end
     a = -3.1
     @test x ⊆ (x^a)^(1/a)
 end
+
+@testset "abs2 and abs" begin
+    x = (0..3) + (0..4)*im
+    @test abs2(x) == 0..25
+    @test abs(x) == norm(x) == 0..5
+
+    y = (-1..1) + (-2..2)*im
+    @test abs(y).lo == 0.0
+    @test abs2(y).lo = 0.0
```

Comment:
```suggestion
    @test abs2(y).lo == 0.0
```

The tests fails because of this line, I should have tested it before suggesting.

It's the occasion to test the "suggest change feature".

Reply:
Thanks! I should have checked more carefully (and tested locally).